package.json

```diff
@@ -40,6 +40,8 @@
     "babelify": "^7.2.0",
     "bluebird": "^3.0.6",
     "browserify": "^12.0.1",
+    "lodash": "^3.10.1",
+    "tsify": "^0.13.1",
```

Comment:
I was having issues importing `tsify`, `babelify`, etc if they were not included installed via the plugin itself.
That being said I just remembered that plugins all share a single `node_modules`. I'll play around with it and report back!